From d2fccf7fc5611cfd60fbe7a8e0bd35d665ac00f7 Mon Sep 17 00:00:00 2001
From: Hood Chatham <roberthoodchatham@gmail.com>
Date: Thu, 31 Mar 2022 17:24:56 -0700
Subject: [PATCH 11/14] not build lapack lite as 64-bit

This patch is to avoid adding macros which will build lapack-lite in  64 bit integer mode, which is not compatible with wasm's 32 bit pointers.
---
 numpy/linalg/setup.py | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/numpy/linalg/setup.py b/numpy/linalg/setup.py
index 94536bb2c..df6c61b9f 100644
--- a/numpy/linalg/setup.py
+++ b/numpy/linalg/setup.py
@@ -37,14 +37,14 @@ def configuration(parent_package='', top_path=None):
         class numpy_linalg_lapack_lite(system_info):
             def calc_info(self):
                 info = {'language': 'c'}
-                if sys.maxsize > 2**32:
+                # if sys.maxsize > 2**32:
                     # Build lapack-lite in 64-bit integer mode.
                     # The suffix is arbitrary (lapack_lite symbols follow it),
                     # but use the "64_" convention here.
-                    info['define_macros'] = [
-                        ('HAVE_BLAS_ILP64', None),
-                        ('BLAS_SYMBOL_SUFFIX', '64_')
-                    ]
+                    # info['define_macros'] = [
+                    #     ('HAVE_BLAS_ILP64', None),
+                    #     ('BLAS_SYMBOL_SUFFIX', '64_')
+                    # ]
                 self.set_info(**info)
 
         lapack_info = numpy_linalg_lapack_lite().get_info(2)
-- 
2.25.1

